resources/devbox/iis-https/setup-iis-https.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+echo_heading2() {
+  echo ""
+  echo_heading "$*"
+}
+
+get_builder_OS
+
+if [ $os_id != win ]; then
+  fail "This script can only be run on Windows to setup IIS."
+fi
+
+warn "Warning: this script requires Administrator permissions to install certificates."
+
+# Generate CA key
+echo_heading2 "Generating CA key"
+if [ ! -f rootCA.key ]; then
+  echo "Please create a password for the certification authority key"
+  echo "This should be unique to your setup."
+  winpty openssl genrsa -des3 -out rootCA.key 2048
+else
+  echo "Skipping, rootCA.key already present"
+fi
+
+# Generate CA certificate
+echo_heading2 "Generating CA certificate"
+if [ ! -f rootCA.pem ]; then
+  winpty openssl req -x509 -new -nodes -key rootCA.key -sha256 -days 3650 -out rootCA.pem
```

Comment:
Does this script need to be re-run when the certs expire?

Reply:
once every ten years, yep.